Backend/app.js

```diff
@@ -0,0 +1,81 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const recipe = require('./model/model');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+require('dotenv').config();
+
+const app = express();
+
+mongoose
+	.connect(process.env.mongo, {
+		useNewUrlParser: true,
+		useUnifiedTopology: true,
+	})
+	.then(() => console.log('Connexion à MongoDB réussie !'))
+	.catch(() => console.log('Connexion à MongoDB échouée !'));
+
+mongoose.set('useCreateIndex', true);// avoid deprecated message, non essential
+app.use(cors());
+app.use(express.static('public'));
+app.use('/', express.static('public'));
+app.use(bodyParser.json());
+
+app.get('/api', (req, res) => {
+	recipe
+		.find({ recommend: false })
```

Comment:
I would include the popular recipes in the list of all recipes, that would make more sense to me
```suggestion
		.find({ recommend: false })
```